transaction.go

```diff
@@ -115,6 +125,7 @@ func newTransaction(state *State) *Transaction {
 	// create a new transaction.
 	tx := &Transaction{
 		cursor: state,
+		id: rand.Int(),
```

Comment:
the transaction ID should be incrementental like buckets as it makes it easier to read and understand ordering.

Reply:
fixed in https://github.com/linden/tempdb/pull/1/commits/35d0436e398ebee6bbfb246a44451c4b52effc95.